app/views/analytics/facility_groups/show.html.erb

```diff
@@ -0,0 +1,56 @@
+<%= render 'shared/analytics/analytics_period_links' %>
+
+<div class="dashboard">
+
+  <h1><%= @facility_group.name %></h1>
+
+  <div class="featured">
+    <table class="featured-table">
+      <tr>
+        <td class="featured-td">
+          <%= render "shared/analytics/patients_count_panel",
+                     newly_enrolled_patients_count: @facility_group_analytics[:newly_enrolled_patients],
+                     returning_patients_count: @facility_group_analytics[:returning_patients] %>
+        </td>
+
+        <td class="featured-td featured-middle">
+          <%= render "shared/analytics/non_returning_hypertensive_patients_panel",
+                     non_returning_hypertensive_patients_count: @facility_group_analytics[:newly_enrolled_patients],
+                     non_returning_hypertensive_patients_per_month: @facility_group_analytics[:non_returning_hypertensive_patients_per_month] %>
+        </td>
+        <td class="featured-td">
+          <%= render "shared/analytics/control_rate_panel",
+                     goal: 45,
+                     control_rate: @facility_group_analytics[:control_rate][:control_rate],
+                     control_rate_per_month: @facility_group_analytics[:control_rate_per_month] %>
+        </td>
+      </tr>
+    </table>
+  </div>
+</div>
+
+
+<%= render 'facility_table' %>
+
+<section>
+  <h4>Notes</h4>
+  <p class="footnote"><strong>Didn't attend</strong> patients are enrolled hypertensives that have not had a BP recorded
+    in the past 3 months.</p>
+  <p class="footnote">
+    <strong><%= t('analytics.control_rate') %></strong>
+    <%= t('analytics.control_rate_explanation',
+          hypertensive_patients_in_cohort: @facility_group_analytics[:control_rate][:hypertensive_patients_in_cohort],
+          patients_under_control: @facility_group_analytics[:control_rate][:patients_under_control_in_period],
+          control_rate_for_month: @facility_group_analytics[:control_rate][:control_rate],
+          number_of_months: pluralize(1, 'month')) %>
+  </p>
+  <p class="footnote"><strong>Top performers:</strong> > 100 enrolled patients and highest BP control.</p>
```

Comment:
Minor space issue: before ` >` after `</strong>`

Reply:
Changed `>` to `&gt;` to be more explicit.